docs/Upgrading.md

```diff
@@ -70,7 +70,7 @@ var Person = MediaType("application/vnd.goa.person", func() {
 		Attribute("age", Integer, "Age of person")
 		Required("name")
 	})
-	View("default", func() { // View defines a rendering of the media type.
+	View("default", func() {  // View defines a rendering of the media type.
 		Attribute("id")   // Media types may have multiple views and must
 		Attribute("href") // have a "default" view.
```

Comment:
should these actually just be `name` and `age`? Not sure where `id` and `href` come from.

Reply:
It must be `name` and `age`. Good catch!

Reply:
Fixed 👍